src/app/Main.js

```diff
@@ -11,9 +11,18 @@ import { serverApiRecordEvent } from 'app/utils/ServerApiClient';
 import * as steem from '@steemit/steem-js';
 import { determineViewMode } from 'app/utils/Links';
 
-window.onerror = error => {
-    if (window.$STM_csrf) serverApiRecordEvent('client_error', error);
-};
+window.addEventListener('error', error => {
+    const trace =
+        typeof error.error === 'object' && typeof error.error.stack === 'string'
```

Comment:
It looks like the ErrorEvent.error is an experimental part of the API ... 
Might need an additional condition to poly-fill for it,
for example if I throw a simple string like `throw 'this is just a test'`  there is no `error` object... and this breaks.

Reply:
it shouldn't break; it should just define the stacktrace property as `false`. unfortunately there is no way to polyfill a stacktrace.

Reply:
Like you need to check that the errorEvent has that object... per docs
![error_msg](https://user-images.githubusercontent.com/3374538/34397255-90a343f8-eb42-11e7-8bab-3e0b778f140c.png)

Reply:
Ey, actually `typeof error.error === object` probably should suffice ... it's just if it is `errorEvent.error` is object and does not have a stack atttribute it will break here.

Reply:
Ok so the thing here is that for some errors the `ErrorEvent.error` is `null` and in this case the double check for: `typeof error.error === 'object' && typeof error.error.stack === 'string'` is insufficient...
demo in console...
```
const e = null
const p = typeof e === 'object' // evals to true
const q = typeof e === 'object' && typeof e.nonExistant === 'string' ? "hello" : "world"
```

Reply:
which errors would have a null error?

i've added the test but from what i understand the prop just won't be present if it isn't supported.

here's a test i ran in ie10:

![screenshot from 2017-12-28 09-18-20](https://user-images.githubusercontent.com/99194/34413268-3b71b770-ebb0-11e7-8bb4-800885abfbff.png)

Reply:
Try:
```
throw "This is a string"
```

Reply:
that's a funny way to throw an error ;) anyways i've added a `null` check

Reply:
![screenshot from 2017-12-28 09-49-30](https://user-images.githubusercontent.com/3374538/34413943-8020a7c4-ebb4-11e7-85b9-fc5d9267d577.png)

Reply:
You can fling strings